drivers/Keypad-PCF8574/Keypad/KeyPressedEventArgs.cs

```diff
@@ -19,6 +19,11 @@ public class KeyPressedEventArgs
         /// Row index of the pressed key.
         /// </summary>
         public sbyte Row => _row;
+        
+        /// <summary>
+        /// Key selected if any is defined by keymap.
+        /// </summary>
+        public char Key { get; set; }
 
```

Comment:
This is more like a formal issue. This should be settable in the constructor only. Like the other ones.
Thus preventing unwanted changes. After all the only place where this should be set is in the constructor along with the row&column.